### DISCUSSION TITLE
Best sampling strategy for opentelemetry & jaeger architecture

### DISCUSSION THREAD

**Discussion by coder-1111** [open-telemetry/opentelemetry-collector#3786] [Q&A] [answered]
Hi,

Iam working on setting up a tracing tool. The data flow would be - application -> otel-collector-> jaeger.
I want to enable sampling in my setup. These are the ways i have explored - 
1.  Sampling from application side - parentbased_traceidratio
2.  Sampling on otel-collector level - probabilisticsamplerprocessor (which would work same as 1st point)
3. I'm not sure what can be done from jaeger side

I have explored about tail based sampling. I see that there is a processor for that usecase on otel-contrib repo - https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/processor/tailsamplingprocessor. But it doesn't support multiple instances of otel-collector. 

Please suggest me a best way to implement sampling for my setup
**Note- I'm using autoinstrumentation**

**Comment by bogdandrutu** [comment, marked as answer]
Duplicate of https://github.com/open-telemetry/opentelemetry-collector-contrib/discussions/4462
